WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/quicklinkribbons/QuickLinkRibbonViewHolder.kt

```diff
@@ -4,36 +4,52 @@ import android.annotation.SuppressLint
 import android.view.GestureDetector
 import android.view.MotionEvent
 import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.OnItemTouchListener
 import org.wordpress.android.databinding.QuickLinkRibbonListBinding
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.QuickLinkRibbon
 import org.wordpress.android.ui.mysite.MySiteCardAndItemViewHolder
-import org.wordpress.android.ui.utils.UiHelpers
 import org.wordpress.android.util.extensions.viewBinding
 
-private const val Y_BUFFER = 100
+private const val Y_BUFFER = 10
 
 class QuickLinkRibbonViewHolder(
-    parent: ViewGroup,
-    private val uiHelpers: UiHelpers
+    parent: ViewGroup
 ) : MySiteCardAndItemViewHolder<QuickLinkRibbonListBinding>(
         parent.viewBinding(QuickLinkRibbonListBinding::inflate)
 ) {
+    init {
+        with(binding.quickLinkRibbonItemList) {
+            if (adapter == null) {
+                layoutManager = LinearLayoutManager(context, RecyclerView.HORIZONTAL, false)
+                val readerInterestAdapter = QuickLinkRibbonItemAdapter()
+                adapter = readerInterestAdapter
```

Comment:
@AjeshRPai - Probably want to rename this something other than `readerInterestAdapter` 😉